lib/mi_bridges/driver/base_page.rb

```diff
@@ -33,7 +33,25 @@ def close
 
       def click_id(id)
         id = "##{id}" unless id.include?("#")
-        page.execute_script("$('#{id}').click()")
+        js_click_selector(id)
+      end
+
+      def js_click_selector(selector)
+        page.execute_script %($("#{selector}").click())
+      end
+
+      def check_in_section(section, options = {})
+        predicate = options.fetch(:if, false)
+        name = options.fetch(:for, "")
```

Comment:
Why use an options hash instead of named parameters? Because `if` is a reserved keyword, but `if` as a hash key is 👍, 👌, no problemo.

Reply:
woah! good context